Specs/Scene/ShadowMapSpec.js

```diff
@@ -50,8 +50,12 @@ describe(
     const floorHeight = -1.0;
```

Comment:
The PR description mentioned that a few specs are breaking, but I don't see that when I run it locally, was the comment out-of-date?

Reply:
The PR description says these specs are breaking when you replace `boxUrl` with `boxExperimentalUrl` completely. That's why I had to keep both models around.